tests/end_to_end_test/restore_test.go

```diff
@@ -488,6 +493,266 @@ func TestRestoreSnapshotOfSingleFile(t *testing.T) {
 	verifyFileMode(t, filepath.Join(restoreDir, "restored-5"), defaultRestoredFilePermission)
 }
 
+func TestSnapshotSparseRestore(t *testing.T) {
```

Comment:
Currently failing in CI on Ubuntu

Reply:
The following (and similar) lines are failing:
```go
	sources := clitestutil.ListSnapshotsAndExpectSuccess(t, e)
```
with output:
```log
snapshot_create_test.go:74: finished in 337: 'kopia --config-file /tmp/kopia-test1214146990/fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff/.kopia.config --no-use-keyring snapshot list -l --manifest-id'
    snapshot_create_test.go:74: can't parse source info: "\x1b[m  2022-03-18 19:08:39 UTC k75283678450059522bca4aaa551375ac 54.8 KB drwx------ manifest:3810e2da6d7d38ee3e28f778addfa32c files:1151 dirs:211 (latest-2)"
```
This issue also coming up on latest master on other snapshot related tests (i.e `TestSnapshotCreate`), not just the ones introduced in this PR. The culprit is the ANSI reset code `\x1b[m` at the beginning of the line–the function `MustParseSnapshots()` mistakes it for a valid source string and tries to parse it. We should raise a separate issue for this.

Reply:
Fixed in the latest push.